tensornetwork/linalg/tests/test_operations.py

```diff
@@ -0,0 +1,326 @@
+# Copyright 2019 The TensorNetwork Authors
```

Comment:
add tests for `SymmetricBackend` where applicable

Reply:
If we want the symmetric backend to be tested by default, wouldn't it make more sense to add this to the pytest config

Reply:
I'd love to do that. The annoying thing is that tensor initialization is so different, and we need to modify basically all existing tests

Reply:
Fair enough. There is still the basic problem that I also have no idea how to initialize symmetric arrays, what their shapes mean, etc